apps/coscrad-frontend/src/components/resources/spatial-features/spatial-feature-index.presenter.css

```diff
@@ -0,0 +1,3 @@
+.leaflet-container {
+    height: 400px;
```

Comment:
This may change a lot with the design of the site.  For example, if we wanted a fullscreen map.  Or if we want to use the map as part of a geographic selector component in a form like select your home depot store location.  It might be better to have this as an inline style in the component itself.